### PR TITLE
added IsComputerControlled manipulation for metrics

### PR DESCRIPTION
**Pull request by jannast**
added IsComputerControlled manipulation for metrics

# Checklist

The following aspects have been respected by the author of this pull request, confirmed by both pull request assignee **and** reviewer:

* Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* README update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer

### CODE REVIEW

File: src/t2iapi/metric/metric_responses.proto [hunk 1]
```diff
@@ -40,3 +40,9 @@ message MetricValueResponse {
     string step_width = 3;  // string representation of the decimal value of the metric value step width
     PartialCodedValue unit = 4;  // the metric unit, represented by a partial coded value with translation
 }
+
+// Response containing the answer to the question if the metric is computer controlled
```
Comment (ldeichmann) [inline src/t2iapi/metric/metric_responses.proto:44]:
```suggestion
// Response indicating whether a metric is computer-controlled
```

File: src/t2iapi/metric/service.proto [hunk 2]
```diff
@@ -100,4 +100,10 @@ service MetricService {
     AbstractMetricState/@ActivationState, but the internal state of the device.
      */
     rpc SetMetricStatus (t2iapi.metric.SetMetricStatusRequest) returns (BasicResponse);
+
+    /*
+    Requests if the given metric handle is a computer controlled metric.
```
Comment (ldeichmann) [inline src/t2iapi/metric/service.proto:105]:
```suggestion
    Requests if the given metric handle is a computer controlled metric.
    
    A computer-controlled metric is a metric with @MetricCategory ∈ { Set, Preset } which is automatically adapted by the device under test, e.g. for an FiO2 closed-loop within a device.
```

File: src/t2iapi/metric/metric_responses.proto [hunk 3]
```diff
@@ -40,3 +40,9 @@ message MetricValueResponse {
...
+message IsComputerControlledResponse {
+    BasicResponse status = 1;  // the status of the rpc indicating whether or not it was successful
+    bool answer = 2;  // True if metric is computer controlled, otherwise False
```
Comment (ldeichmann) [inline src/t2iapi/metric/metric_responses.proto:47]:
```suggestion
    bool answer = 2;  // true if metric is computer-controlled, otherwise false
```

**Review by ldeichmann — CHANGES_REQUESTED** [review]

File: src/t2iapi/metric/service.proto (hunk 2 above)
[the remark this answers is not in the document]
Comment (jannast) [reply src/t2iapi/metric/service.proto]: done

File: src/t2iapi/metric/metric_responses.proto (hunk 1 above)
Comment (jannast) [reply src/t2iapi/metric/metric_responses.proto]: done

File: src/t2iapi/metric/service.proto (hunk 2 above)
Comment (jannast) [reply src/t2iapi/metric/service.proto]: done

File: src/t2iapi/metric/metric_responses.proto (hunk 3 above)
Comment (jannast) [reply src/t2iapi/metric/metric_responses.proto]: done

**Review by a-kleinf — APPROVED** [review]

**Review by ldeichmann — APPROVED** [review]

**Pull request merged**